cmd/ddev/cmd/describe.go

```diff
@@ -62,6 +62,11 @@ func renderAppDescribe(desc map[string]interface{}) (string, error) {
 	ddevapp.RenderAppRow(appTable, desc)
 	output = fmt.Sprint(appTable)
 
+	output = output + "\n\nSite Information\n-----------------\n"
```

Comment:
Let's go ahead and change "Site" to "Project", since that's the path we've chosen.  One less thing to alter in the other issue.